### DISCUSSION TITLE
View "WILDCARDS" menu/dropdown menu in Fooocus Webui

### DISCUSSION THREAD

**Discussion by tmprabubiz** [lllyasviel/Fooocus#2567] [Q&A] [answered]
Is there any settings in the latest Fooocus to view the wildcards as a menu or dropdown menu in the Fooocus webui. 
Since, I am using  lot of wildcards, remembering all the file names is quite impossible. 
Please guide

**Comment by mashb1t** [comment, marked as answer]
No, this has not (yet) been implemented.

**Reply by tmprabubiz** [reply]
How far I can have faith in "https://github.com/metercai/SimpleSDXL?tab=readme-ov-file"
The full package installation is warned as mallicious by the browser itself.
Please Guide

**Comment by ghost** [comment]
Integrating a feature where typing "__" automatically suggests a list of available wildcards, akin to an autocomplete function, could enhance user experience.

**Reply by mashb1t** [reply]
=> https://github.com/lllyasviel/Fooocus/issues/2585

**Reply by metercai** [reply]
This feature is already available and you will try it here: https://github.com/metercai/SimpleSDXL
Will automatically open the wildcards panel based on triggering characters "[" or "_".
<img width="600" src="https://github.com/lllyasviel/Fooocus/assets/5652458/be76ae6f-97b1-4ba5-9363-2b07fe3720ee">

**Reply by mashb1t** [reply]
@metercai would you mind creating a PR or could the code in your repository be used in Fooocus?

**Reply by metercai** [reply]
I once wanted to create a PR, but I gave up because I found that it was related to other unique modules of SimpleSDXL, making it difficult to merge with the Fooocus code.
